### DISCUSSION TITLE
The internal WPF test infrastructure is now available

### DISCUSSION THREAD

**Discussion by ryalanms** [dotnet/wpf#5544] [Announcements]
The internal WPF test repo is being published as open source.   The new repo is here:  https://github.com/dotnet/wpf-test. 

The internal test infrastructure and test collateral are many times larger than the WPF source itself, and the WPF team will be publishing more internal test collateral to this repo as it goes through scrubbing and open source review.

**Comment by lindexi** [comment]
Awesome!

Does it run on every unit tests?

**Comment by ryalanms** [comment]
@lindexi:  @vishalmsft's team is working on it.
